src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+import java.text.DecimalFormat;
+
+public class IsoscelesTrapezoid extends Figure {
+    private final int side;
+    private final int topSide;
+    private final int downSide;
+    private final double height;
+    private final DecimalFormat format;
+
+    public IsoscelesTrapezoid(String color, int side, int topSide, int downSide) {
+        super(color);
+        this.format = new DecimalFormat("#0.0");
```

Comment:
If it is a constant you can make it static, to avoid creating redundant instances